src/common/fileUpload/fileUploadButton/fileUploadButton.module.scss

```diff
@@ -0,0 +1,139 @@
+.button {
+  cursor: pointer;
+  font-family: var(--hds-theme-primary-font);
+  font-weight: 500;
+  padding: 0.875em 0.7em;
+  transition-property: background-color, border-color;
+  transition-duration: 0.15s;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  border: 4px solid rgba(0, 0, 0, 0);
+  fill: currentColor;
+  text-align: left;
+  align-content: flex-start;
+  vertical-align: top;
+}
+
+.button:focus {
+  background-clip: padding-box;
+}
+
+.button:after {
+  content: '';
+  display: block;
+  position: absolute;
+  top: -4px;
+  left: -4px;
+  right: -4px;
+  bottom: -4px;
+  border: 2px solid rgba(0, 0, 0, 0);
+}
+
+.disabled {
+  pointer-events: none;
+}
+
+.label {
+  margin: 0 1.375em;
+}
+
+.primary {
+  background-color: var(--hds-theme-color-primary);
+  color: var(--hds-theme-color-primary-heading);
+}
+
+.primary:after {
+  border-color: var(--hds-theme-color-primary);
+}
+
+.primary.disabled {
+  background-color: var(--hds-theme-color-primary-light);
+}
+
+.primary.disabled:after {
+  border-color: var(--hds-theme-color-primary-light);
+}
+
+.primary:hover {
+  background-color: var(--hds-theme-color-primary-dark);
+}
+
+.primary:hover:after {
+  border-color: var(--hds-theme-color-primary-dark);
+}
+
+.secondary {
+  background-color: var(--hds-theme-color-secondary);
+  color: var(--hds-theme-color-secondary-heading);
+}
+
+.secondary:after {
+  border-color: var(--hds-theme-color-secondary);
+}
+
+.secondary.disabled {
+  background-color: var(--hds-theme-color-secondary-light);
+}
+
+.secondary.disabled:after {
+  border-color: var(--hds-theme-color-secondary-light);
+}
+
+.secondary:hover {
+  background-color: var(--hds-theme-color-secondary-dark);
+}
+
+.secondary:hover:after {
+  border-color: var(--hds-theme-color-secondary-dark);
+}
+
+.tertiary {
+  background-color: var(--hds-theme-color-tertiary);
+  color: var(--hds-theme-color-tertiary-heading);
+}
+
+.tertiary:after {
+  border-color: var(--hds-theme-color-tertiary);
+}
+
+.tertiary.disabled {
+  background-color: var(--hds-theme-color-tertiary-light);
+}
+
+.tertiary.disabled:after {
+  border-color: var(--hds-theme-color-tertiary-light);
+}
+
+.tertiary:hover {
+  background-color: var(--hds-theme-color-tertiary-dark);
+}
+
+.tertiary:hover:after {
+  border-color: var(--hds-theme-color-tertiary-dark);
+}
+
+.supplementary:after {
+  border-color: currentColor;
+}
+
+.supplementary:focus:after {
+  border-width: 4px;
+}
+
+.supplementary.disabled {
+  color: var(--hds-ui-color-black-30);
+}
+
+.supplementary.disabled:after {
+  border: var(--hds-ui-color-black-30);
+}
+
+.supplementary:hover {
+  text-decoration: underline;
+}
+
+.small {
+  padding: 0.25em;
+}
```

Comment:
This is copy-pasted from HDS

Reply:
But using HDS custom variables in our SASS modules will make it hard to trace and update in the future, especially if they introduced some breaking changes as they did since the absence of a CSS variable's value would fail silently. 
That's why we mapped all of their values to our own SASS variables in one partial `_colours.scss`, so when they update their variables we know that we need only to update it in one place.
[These changes](https://github.com/City-of-Helsinki/berth-reservations-admin/pull/188/files#diff-eddb3742e0a6f751810131793ef6a720) in @santtul's PR may explain my point.